src/Uno.UI/FeatureConfiguration.cs

```diff
@@ -111,6 +111,17 @@ public static class ListViewBase
 			public static double? DefaultCacheLength = 1.0;
 		}
 
+#if __ANDROID__
+		public static class NativeListViewBase
+		{
+			/// <summary>
+			/// Sets this value to remove item animation for <see cref="UnoRecyclerView"/>. This prevents <see cref="UnoRecyclerView"/>
+			/// from crashing when pressured: Tmp detached view should be removed from RecyclerView before it can be recycled
+			/// </summary>
+			public static bool RemoveItemAnimator = true;
```

Comment:
@MatFillion @XiaotianNetlift So... by default no more ListView insert/remove animations on Android? That's not going to be a regression for somebody?

Reply:
@davidjohnoliver I created #739